src/lean_spec/subspecs/forkchoice/store.py

```diff
@@ -957,6 +949,7 @@ def produce_block_with_signatures(
         # Get parent block and state to build upon
         store, head_root = self.get_proposal_head(slot)
         head_state = store.states[head_root]
+        post_state = head_state
```

Comment:
This `post_state` gets replaced at https://github.com/leanEthereum/leanSpec/pull/212/files#diff-67c42b82824779fcb11cc28349098aa13e096ed5a4b759919fce36a185ffcbe4R981 so this line is redundant